Source/WebCore/page/NavigationDestination.cpp

```diff
@@ -32,4 +32,11 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(NavigationDestination);
 
+NavigationDestination::NavigationDestination(const URL& url, RefPtr<NavigationHistoryEntry> entry, bool isSameDocument)
```

Comment:
`RefPtr<NavigationHistoryEntry>&&` or `NavigationHistoryEntry*` but let's not pass a RefPtr by value.